nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -289,6 +289,15 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
   - The `erlang_node_short_name`, `erlang_node_name`, `port` and `options` configuration parameters are gone, and have been replaced with an `environment` parameter.
     Use the appropriate [environment variables](https://hexdocs.pm/livebook/readme.html#environment-variables) inside `environment` to configure the service instead.
 
+- Configuring PipeWire and WirePlumber via `environment.etc` is no longer supported due to limitations of `environment.etc` and how `services.pipewire.configPackages` and `services.pipewire.wireplumber.configPackages` were implemented .
+  - PipeWire configuration should now use `services.pipewire.extraConfig`. Packages providing configuration files can now be specified in `services.pipewire.configPackages`.
+  - WirePlumber is only configurable via `services.pipewire.wireplumber.configPackages` until [#292115](https://github.com/NixOS/nixpkgs/pull/292115) lands. For example, to replace a config file `/etc/wireplumber/main.lua.d/41-set-celluloid-default-volume.lua` please use
+    ```nix
+    pkgs.writeTextDir "share/wireplumber/main.lua.d/41-set-celluloid-default-volume.lua" ''
```

Comment:
> main.lua.d

Is this still the case in wireplumber 0.5? https://pipewire.pages.freedesktop.org/wireplumber/daemon/configuration/conf_file.html currently says the following:

> Starting with WirePlumber 0.5, this is the only file that WirePlumber reads to load configuration (together with its fragments - see below). In the past, WirePlumber also used to read Lua configuration files that were referenced from wireplumber.conf and all the heavy lifting was done in Lua. This is no longer the case, and the Lua configuration files are no longer supported.

and I don't see any mention of `main.lua.d` in the documentation. (you are probably already well aware of this, but I figure I'd mention it on this PR explicitly)

Reply:
No, this is no longer accurate. I'll adjust this for 0.5 in the next days. See also #292115